home/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestModels:
+
+    def test_add_profile(self):
+        profile = self.get_profile()
```

Comment:
please turn `get_profile` into a fixture (it could just be called `profile` then)